src/App/Pages/Generator/GeneratorPageViewModel.cs

```diff
@@ -564,26 +567,28 @@ public bool IncludeNumberRandomWordUsername
 
         public UsernameEmailType PlusAddressedEmailTypeSelected
         {
-            get => _plusAddressedEmailTypeSelected;
+            get => _usernameOptions.PlusAddressedEmailType;
             set
             {
-                if (SetProperty(ref _plusAddressedEmailTypeSelected, value))
+                if (_usernameOptions.PlusAddressedEmailType != value)
                 {
                     _usernameOptions.PlusAddressedEmailType = value;
-                    SaveUsernameOptionsAsync(false).FireAndForget();
+                    TriggerPropertyChanged(nameof(PlusAddressedEmailTypeSelected));
+                    SaveUsernameOptionsAsync().FireAndForget();
                 }
             }
         }
 
         public UsernameEmailType CatchAllEmailTypeSelected
         {
-            get => _catchAllEmailTypeSelected;
```

Comment:
Remove the field given that it's no longer used

Reply:
Field removed.